### ISSUE TITLE
Wetransfer to alternatives redirect

### ISSUE THREAD

**Issue by tarbib** [SimonBrazell/privacy-redirect#211]
Hello,

Would you consider a new type of redirect?

Like wetransfer.com to upload.disroot.org

Thanks

**Comment by TROMsite** [comment]
Could be cool. We also have such an instance https://send.trom.tf/

**Comment by alefvanoon** [comment]
but they are not frontend for wetransfer they are alternative service.

**Comment by tarbib** [comment]
@alefvanoon such as "Search Engine Redirects" feature ;)

**Comment by SimonBrazell** [comment]
I think this would just cause problems with actual Wetransfer links though right, which I think is what @alefvanoon was getting at?

If someone sends me an email with a Wetransfer link in it and I click it, I'd be taken to the alternative site and not receive the file as I intended, would probably be quite frustrating.

**Comment by TROMsite** [comment]
Simon is 100% correct. I don0t think this can be done.

**Comment by SimonBrazell** [comment]
Maybe someone needs to make an alternative Wetransfer front-end! 🙂

**Comment by tarbib** [comment]
Right! I see what you mean now. As the file is stored on wetransfer, redirecting to another service would not let you download it.
My comment was more related to redirecting before uploading as I tend to forget the alternatives to wetransfer.
